### PR TITLE
Added consistencss

### PR DESCRIPTION
**Pull request by mateosilguero**
# Why

Add [consistencss](https://github.com/mateosilguero/consistencss) library.

# Checklist

If you added a new library:

- [x] Added it to **react-native-libraries.json**

If you added a feature or fixed a bug:

- [ ] Documented in this PR how to use the feature or replicate the bug.
- [ ] Documented in this PR how you fixed or created the feature.

### CODE REVIEW

**Review by Simek — CHANGES_REQUESTED** [review]
Hi @mateosilguero, due to compatibility issues in the past, we no longer accept out-of-tree platforms without a working demo or example (even if, from the source code, they are looking like they would work).

Also I was try to import the listed example into Snack, but it failed. 

If you want to see the library accepted quickly (without spending time on writing the examples) unfortunately you need to removed the out-of-tree platforms. Of course you will be able to add platform tags later in time.

File: react-native-libraries.json [hunk 1]
```diff
@@ -6073,5 +6073,20 @@
...
+     ],
+     "images": [
+       "https://raw.githubusercontent.com/mateosilguero/consistencss/master/website/static/img/logo.png"
+     ],
+     "ios": true,
+     "android": true,
+     "web": true,
+     "expo": true,
+     "windows": true,
+     "macos": true
```
Comment (mateosilguero) [inline react-native-libraries.json:6090]:
```suggestion
     "android": true,
     "ios": true
```

**Review by Simek — APPROVED** [review]
Thank you for the review update! 👍

**Comment by mateosilguero** [comment]
Hi @Simek, I understand, i removed the other platforms.

**Pull request merged**